package.json

```diff
@@ -10,6 +10,7 @@
   },
   "devDependencies": {
     "@hpe/project-scripts": "^1.0.8",
+    "eslint-plugin-prettier": "^3.3.0",
```

Comment:
Background on why this was added?

Reply:
This might be able to be removed -- when I was running the pre-commit linting, it wasn't finding eslint-plugin-prettier but it could be an issue with how I had things installed locally. if you're able to remove it and still pass all the pre-commit checks, then we should remove it and I will explore what's happening locally for me.

Reply:
Okay. We can do some investigating.